pydm/widgets/embedded_display.py

```diff
@@ -229,7 +256,7 @@ def open_file(self, force=False):
                     parent_file_path = os.path.realpath(parent_file_path)
                 base_path = os.path.dirname(parent_file_path)
 
-            fname = find_file(self.filename, base_path=base_path, raise_if_not_found=True)
+            fname = find_file(self.filename, base_path=base_path, raise_if_not_found=True, subdir_scan_enabled=True)
```

Comment:
```suggestion
            fname = find_file(self.filename, base_path=base_path, raise_if_not_found=True, subdir_scan_enabled=self._recursive_display_search)
```
I think we want to use the new property here too?

Reply:
Yep, fixed